src/objects/zcl_abapgit_object_intf.clas.locals_imp.abap

```diff
@@ -456,25 +462,104 @@ CLASS lcl_aff_type_mapping IMPLEMENTATION.
     es_data = ls_data_aff.
   ENDMETHOD.
 
+  METHOD zif_abapgit_aff_type_mapping~to_abapgit.
+    DATA:
+      ls_data_abapgit TYPE zcl_abapgit_object_intf=>ty_intf,
+      ls_data_aff     TYPE zif_abapgit_aff_intf_v1=>ty_main.
+
+
+    ls_data_aff = iv_data.
+
+    ls_data_abapgit-description = set_abapgit_descriptions( is_clsname = 'MYCLASS'
```

Comment:
Here we need the actual interface name. Otherwise the SAPGUI description texts are not deserialized properly.